src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -206,8 +211,8 @@ internal void Validate()
 
             if (IsInNonProductionSlot() && this.IsDefaultHubName())
             {
-                throw new InvalidOperationException("Task Hub name must be specified in host.json when using slots. See documentation on Task Hubs for " +
-                    "information on how to set this: https://docs.microsoft.com/azure/azure-functions/durable/durable-functions-task-hubs");
+                throw new InvalidOperationException($"Task Hub name must be specified in host.json when using slots. Specified name must not equal the defaultHubName ({this.defaultHubName})." +
```

Comment:
I think people will be confused when they see `defaultHubName` in this message because that's the name of a some private variable that they won't know anything about. I suggest rewording this without referring to the word "defaultHubName".